File: python/modele/reservation.py
from datetime import datetime

class Reservation:
    def __init__(self, name, email, start_date, end_date):
        self.name = name
        self.email = email
        self.start = start_date
        self.end = end_date

    def display(self):
        print(f"Name: {self.name}, Email: {self.email}, From: {self.start.strftime('%Y-%m-%d')}, To: {self.end.strftime('%Y-%m-%d')}")

File: typescript/modele/reservation.ts
export default class Reservation {
    name: string;
    email: string;
    start: Date;
    end: Date;
  
    constructor(name: string, email: string, start: Date, end: Date) {
      this.name = name;
      this.email = email;
      this.start = start;
      this.end = end;
    }
  
    display(): void {
      console.log(`👤 ${this.name} (${this.email}) - 📅 ${this.start.toDateString()} à ${this.end.toDateString()}`);
    }
  }
  